apps/coscrad-frontend/src/components/AllResources/ResourceInfos.presenter.tsx

```diff
@@ -12,10 +13,11 @@ export const ResourceInfosPresenter: FunctionalComponent<
 > = ({ data: resourceInfos }: HasData<IAggregateInfo<ResourceType>[]>) => (
     <div>
         <h2>Available Resources</h2>
-        <div>
-            {resourceInfos.map((info) => (
-                <ResourceInfoPresenter {...info} key={info.type}></ResourceInfoPresenter>
-            ))}
-        </div>
+        <Divider />
+        <br />
```

Comment:
Remove <br /> 's and use styling for adding space

Reply:
I'll leave this to you and Justin on the "consistent styling" sweep.

Reply:
But I did remove the `<br />`s for now.